spec/views/devise/sessions/new.html.slim_spec.rb

```diff
@@ -26,7 +26,7 @@
 
     expect(rendered).
       to have_link(
-        t('links.create_account'), href: sign_up_email_path
+        t('links.create_account'), href: sign_up_email_url(request_id: nil)
```

Comment:
I believe it needs to be `_url` to include params. Plus, I think we're moving to using `_url` exclusively. I would assume this would fail if it had to be `_path`.